library-best-practices/glossary.md

```diff
@@ -66,3 +81,55 @@ Java Dependency Glossary
 - **Linkage-matchable version** (said of a particular version of A in relation
   to all versions of B): There exists some version of B such that the version of
   A and the version of B are linkage-compatible.
+
+
+### Class usage graph
+
+- **Class usage graph**: a possibly cyclic directed graph of references between classes
```

Comment:
I'm not sure the terms related to the graph and reachability are general enough to go into the glossary. They're relevant to the "reachability" notion in the static linkage checker, so maybe should be defined there.

Reply:
And there we can maybe just simply say something like:

Only linkage errors that are reachable through some sequence of method calls, field accesses, and class literal references from the entry point set are reported.

(We will actually report more than that for now because we assume everything in a class is reached when anything is, but we could actually narrow it down to that set later, and that would be ideal.)

Reply:
I would prefer to keep these terms in the glossary in case we reference them outside of the static linkage checker (which I feel is pretty likely). Even if not, I think it's best for all terms to be defined in one place so that they are easier to find.

Reply:
To discuss tomorrow.